src/components/Main/Results/DeterministicLinePlot.tsx

```diff
@@ -229,8 +224,17 @@ export function DeterministicLinePlot({ data, userResult, logScale, showHumanize
                   domain={[tMin, tMax]}
                   tickCount={7}
                 />
-                <YAxis scale={logScaleString} type="number" domain={[1, 'dataMax']} tickFormatter={yTickFormatter} />
-                <Tooltip formatter={tooltipFormatter} labelFormatter={labelFormatter} />
+                <YAxis
+                  scale={logScaleString}
+                  type="number"
+                  domain={[1, 'dataMax']}
+                  tickFormatter={(tick) => t('localized:number', { value: tick })}
```

Comment:
This doesn't look like it's accounting for the showHumanized boolean, i.e show as 1.67M rather thatn 1670000